app/views/checkouts/new.html.erb

```diff
@@ -31,7 +31,7 @@
   </div>
 </div>
 
-<script src="https://js.braintreegateway.com/web/dropin/1.9.0/js/dropin.min.js"></script>
+<script src="https://js.braintreegateway.com/web/dropin/1.9.1/js/dropin.min.js"></script>
```

Comment:
Please revert this change. We have an automated script we run to handle this.

Reply:
Ok, I didn't know about this.